MAUI/DataGrid/data-binding.md

```diff
@@ -0,0 +1,116 @@
+
+Below codes demonstrate that how to bind a data source to the `SfDataGrid`. Each row in the SfDataGrid would bind to an object in data source. Each column would bind to a property in the data model object.
+
+{% tabs %}
+{% highlight xaml %}
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:GettingStarted"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.DataGrid;assembly=Syncfusion.Maui.DataGrid"
+             x:Class="GettingStarted.MainPage">
+
+    <ContentPage.BindingContext>
+        <local:ViewModel/>
+    </ContentPage.BindingContext>
+
+    <syncfusion:SfDataGrid x:Name="sfDataGrid" ItemsSource="{Binding OrderInfoCollection}" />
+</ContentPage>
+{% endhighlight %}
+
+{% highlight c# %}
+OrderInfoViewModel orderInfoViewModel = new OrderInfoViewModel();
+this.sfDataGrid.ItemsSource = orderInfoViewModel.OrderInfoCollection;
+{% endhighlight %}
+{% endtabs %}
+
+If the data source implements [ICollectionChanged](https://docs.microsoft.com/en-us/dotnet/api/system.collections.icollection?view=net-6.0) interface, then the `SfDataGrid` will automatically refresh the view when an item is added, removed, or cleared. When you add or remove an item in [ObservableCollection](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.observablecollection-1?view=net-6.0), it automatically refreshes the view as the ObservableCollection. That implements the [INotifyCollectionChanged](https://docs.microsoft.com/en-us/dotnet/api/system.collections.objectmodel.observablecollection-1.system-componentmodel-inotifypropertychanged-propertychanged?view=net-6.0). But when you do the same in `List`, it will not refresh the view automatically.
+
+If the data model implements the `INotifyPropertyChanged` interface, then the SfDataGrid responds to the property change at runtime to update the view.
+
+## Binding with IEnumerable
+
+The `SfDataGrid` control supports to bind any collection that implements from [IEnumerable](https://docs.microsoft.com/en-us/dotnet/api/system.collections.ienumerable?view=net-6.0) interface. All the data operations such as sorting and filtering are supported when binding collection derived from IEnumerable.
+
+## Binding with DataTable
+
+The `SfDataGrid` control supports to bind the [DataTable](https://docs.microsoft.com/en-us/dotnet/api/system.data.datatable?view=net-6.0). `SfDataGrid` control automatically refresh the UI when binding DataTable as ItemsSource when rows are added, removed or cleared.

```

Comment:
The `SfDataGrid` control supports binding the [DataTable](https://docs.microsoft.com/en-us/dotnet/api/system.data.datatable?view=net-6.0). `SfDataGrid` control automatically refreshes the UI when binding DataTable as ItemsSource when rows are added, removed, or cleared.

Reply:
Modified.